pattern-filter/apps/pattern-filter/src/main/java/com/redhat/training/transform/Application.java

```diff
@@ -0,0 +1,203 @@
+				false,
+				new Customer(
+					"Jess",
+					"Thompson",
+					"jthompson",
+					"password",
+					"jthompson@email.com",
+					false
+				)
+			),
+			true
+		);
+		thread(
+			new OrderProducer(
+				new Integer(5),
+				new BigDecimal("0.012"),
+				false,
+				new Customer(
+					"Tony",
+					"Anderson",
+					"tanderson",
+					"password",
+					"tanderson@email.com",
+					false
+				)
+			),
+			true
+		);
+
+	}
+
+	public static OrderItem getOrderItem(int id, int qty, float price, CatalogItem item) {
+		OrderItem o = new OrderItem();
+		o.setId(new Integer(id));
+		o.setQuantity(new Integer(qty));
+		o.setExtPrice(new BigDecimal(price));
+		o.setItem(item);
+		return o;
+	}
+
+	public static void thread(Runnable runnable, boolean daemon) {
+		Thread brokerThread = new Thread(runnable);
+		brokerThread.setDaemon(daemon);
+		brokerThread.start();
+	}
+
+	public static class OrderProducer implements Runnable {
+		Order o = new Order();
+
+		OrderProducer(Integer Id, BigDecimal discount, Boolean delivered, Customer customer) {
+			CatalogItem ci1 = new CatalogItem("ci1", "catalog item 1", new BigDecimal("5"),
+			"catalog item 1 description", "ci1 author", "path/to/ci1.png",
+			"category ci1", true);
+
+			CatalogItem ci2 = new CatalogItem("ci2", "catalog item 2", new BigDecimal("10"),
+			"catalog item 2 description", "ci2 author", "path/to/c21.png",
+			"category ci2", true);
+
+			OrderItem oi1 = getOrderItem(1, 2, 10.0f , ci1);
+			OrderItem oi2 = getOrderItem(2, 3, 30.0f , ci2);
+
+			o.getOrderItems().add(oi1);
+			o.getOrderItems().add(oi2);
+			o.setId(Id);
+			o.setDiscount(discount);
+			o.setDelivered(delivered);
+			o.setCustomer(customer);
+			customer.getOrders().add(o);
+		}
+
+		public void run() {
+			try {
+                // Create a ConnectionFactory
+				ActiveMQConnectionFactory cf = new ActiveMQConnectionFactory();
+        		cf.setBrokerURL("tcp://localhost:61616");
+        		cf.setUser("admin");
+        		cf.setPassword("admin");
+
+                // Create a Connection
+                Connection connection = cf.createConnection();
+                connection.start();
+
+                // Create a Session
+                Session session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
+
+                // Create the destination (Topic or Queue)
+                Destination destination = session.createQueue("orderInput");
+
+				// create the object message
+				ObjectMessage msg = session.createObjectMessage();
+/* 				CatalogItem ci1 = new CatalogItem("001", "CI 1 title", new BigDecimal("20.34"),
+					"First catalog item", "CI author", "catalogitemRoot/images/ci1image","category 1", true); */
+				msg.setObject(o);
+
+                // Create a MessageProducer from the Session to the Topic or Queue
+                MessageProducer producer = session.createProducer(destination);
+                producer.setDeliveryMode(DeliveryMode.NON_PERSISTENT);
+
+				// send the message
```

Comment:
Please fix the spacing and also if all the comment messages start with a capital letter, this should follow that rule.